sympy/solvers/tests/test_solvers.py

```diff
@@ -1792,3 +1793,17 @@ def test_issue_2840_8155():
 
 def test_issue_9567():
     assert solve(1 + 1/(x - 1)) == [0]
+
+def test_issue_11538():
+    assert solve(x + E) == [-E]
+    assert solve(x**2 + E) == [-I*sqrt(E), I*sqrt(E)]
+    assert solve(x**3 + 2*E) == [
+        -cbrt(2 * E),
+        cbrt(2)*cbrt(E)/2 - cbrt(2)*sqrt(3)*I*cbrt(E)/2,
+        cbrt(2)*cbrt(E)/2 + cbrt(2)*sqrt(3)*I*cbrt(E)/2]
+    assert solve([x + 4,y + E],x,y) == {x:-4,y:-E}
+    assert solve([x**2 + 4,y + E],x,y) == [(-2*I, -E), (2*I, -E)]
+
+    e1 = x - y**3 + 4
+    e2 = x + y + 4 + 4 * E
+    assert solve([e1,e2],x,y) != []
```

Comment:
I believe the result is correct, but, actually, the purpose of the test is to show that a result is obtained. Hence it should not be necessary to copy the long answer. I think that something like `assert len(solve([e1, e2], x, y)) == 3` (with a space after the commas) would suffice.